fast/stages-multitenant/1-resman-tenant/templates/workflow-gitlab.yaml

```diff
@@ -13,8 +13,14 @@
 # limitations under the License.
 
 default:
+  id_tokens:
```

Comment:
Actually, have you tested this?
I don't think this works at all under default. Instead it needs to be put in a pipeline step.
I'm working on it and I might have a working version soon but if you have something earlier, please commit it here.